src/Symfony/Component/ExpressionLanguage/Node/NullCoalescedNameNode.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\ExpressionLanguage\Node;
+
+use Symfony\Component\ExpressionLanguage\Compiler;
+
+/**
+ * @author Fabien Potencier <fabien@symfony.com>
```

Comment:
You are the author

Reply:
@OskarStark Yeah I wasn't sure what the customs around the author stuff were, and every file I opened had as author Fabien